pallets/parachain-staking/rpc/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+//! Runtime API definition for Parachain Staking.
+
+#![cfg_attr(not(feature = "std"), no_std)]
+use parity_scale_codec::{Codec, Decode, Encode, MaxEncodedLen};
+use frame_support::scale_info::TypeInfo;
+use frame_support::sp_runtime::Perquintill;
+use sp_std::fmt::Debug;
+use module_oracle_rpc_runtime_api::BalanceWrapper;
```

Comment:
Maybe it's not a good idea to use the type defined in another rpc runtime api here. I'm not sure if we even still need something like the `BalanceWrapper`. Seems like [KILT does not](https://github.com/KILTprotocol/kilt-node/blob/ba088c2bd3a40f2983ed884e58f3a0046eb529df/runtime-api/staking/src/lib.rs#L41), so maybe we can also live without it?

Reply:
I had the understanding given [this comment](https://github.com/pendulum-chain/spacewalk/blob/main/pallets/oracle/rpc/runtime-api/src/lib.rs#L13) in oracle pallet in spacewalk, that it was needed if we ever wanted to set balance as u128. Do you think that the issue is outdated?

Reply:
I can confirm the the return value can be fetched. It simply returns `{amount: balance}` where balance is represented as a string as expected. 
The only caveat is that I couldn't test this using `polkadot.js` since I am having trouble calling the rpc (any custom rpc), but this is a different problem.

Reply:
Alright, thanks for testing it 👍